app/models/album.rb

```diff
@@ -0,0 +1,6 @@
+class Album < ActiveRecord::Base
+  validates :name, presence: true
```

Comment:
Good job doubling up on your validations (model and form). It's a good habit to have